composer.json

```diff
@@ -1,6 +1,6 @@
 {
-    "name": "deepak/pwa-extension",
-    "require": {
+    "name": "rtCamp/pwa-extension",
+    "require-dev": {
         "wpackagist-plugin/pwa": "*"
```

Comment:
Also, we should lock the PWA plugin's version here to the latest version we have tested with. Future changes can break our plugin's features.

Reply:
The composer dependency added here is only for unit testing and it won't break the plugin features. We should run the unit tests with the latest version of the plugin so that we can know if the plugin breaks in the future version of the PWA plugin.